Sources/StreamChat/APIClient/Endpoints/BlockedUserPayload.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// An object describing the incoming blocking user JSON payload.
+struct BlockingUserPayload: Decodable {
+    private enum CodingKeys: String, CodingKey {
+        case blockedUserId = "blocked_user_id"
+        case blockedByUserId = "blocked_by_user_id"
+        case createdAt = "created_at"
+    }
+
+    let blockedUserId: String
+    let blockedByUserId: String
+    let createdAt: Date
+}
+
+extension BlockingUserPayload: Equatable {
+    static func == (lhs: BlockingUserPayload, rhs: BlockingUserPayload) -> Bool {
+        lhs.blockedUserId == rhs.blockedUserId
+            && lhs.blockedByUserId == rhs.blockedByUserId
+            && lhs.createdAt == rhs.createdAt
+    }
```

Comment:
That is not needed since Swift will synthesise this automatically. We sometimes have custom equality implementations when we want to exclude some properties from comparison.

Reply:
Done 👍

Reply:
Done 👍